package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slashauth/slashauth-react",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
I think this would be a PATCH update of version since we are not adding a new feature

Reply:
I generally would agree except we're removing a function so I'm going to make it a minor update.

Generally this could be a major update but given that we're still in beta I'm comfortable with minor changes having breaking updates.